integ-test/src/test/java/org/opensearch/sql/legacy/CsvFormatResponseIT.java

```diff
@@ -719,6 +728,12 @@ private void verifyFieldOrder(final String[] expectedFields) throws IOException
     verifyFieldOrder(expectedFields, query);
   }
 
+  private void assertContainsSameItems(List<String> expectedLines, List<String> actualLines) {
```

Comment:
I think sort and `assertEquals(expectedLines.size(), lines.size());` should be in this method. (sorting is part of verifying same items are in the list regardless of its order)